manifests/s/solidclouds/starborne/1.3.153/solidclouds.starborne.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: solidclouds.starborne
```

Would be better if the casing here matched the publisher and package name
```suggestion
PackageIdentifier: SolidClouds.Starborne
```